pkg/uki/reset.go

```diff
@@ -55,6 +58,19 @@ func (r *ResetAction) Run() (err error) {
 		}
 	}
 
+	// REMOUNT /efi as RW (its RO by default)
+	umount, err := e.MountRWPartition(r.spec.Partitions.EFI)
```

Comment:
maybe just change the name of the var to `mount` instead of `umount`

Reply:
That's really the *unmount* function. It's returned by the `MountRWPartition` : https://github.com/kairos-io/kairos-agent/blob/8c353a95120ab6bc1baf8344164a3905c2bfd18d/pkg/elemental/elemental.go#L167

Reply:
gotcha, thanks for clarifying